hangups/conversation.py

```diff
@@ -11,34 +12,62 @@
 
 
 @asyncio.coroutine
-def build_user_conversation_list(client):
+def build_user_conversation_list(client, conversations_per_request=100):
```

Comment:
Do you think there's a good reason to add `conversations_per_request` or could it just be hard coded?